modules/n1ql/pages/n1ql-language-reference/join.adoc

```diff
@@ -665,11 +666,10 @@ This join hint instructs the planner to use nested-loop join (NL join) for the j
 [source,N1QL]
 ----
 SELECT COUNT(1) AS Total_Count
-FROM `travel-sample` rte
-INNER JOIN `travel-sample` aline
+FROM `travel-sample`.inventory.route rte
```

Comment:
I noticed in some of the other n1ql examples you omitted the alias name, and just used `route` which I preferred
(and which feels like a happy feature of the scope rewrite).

Reply:
My approach was, if the original example applied an alias, leave it as is, even if unnecessary. However,  if the original example applied an alias which had the same name as the collection, e.g. `route` or `airline`, I omitted the alias in the rewrite. By default, the query service automatically applies an alias which is the same as the name of the keyspace. So no need to assign it explicitly.